### PR TITLE
6

### PR DESCRIPTION
**Pull request by edeharrison**

### CODE REVIEW

File: db/data/test-data/playground.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+\c nc_news_test
```
Comment (Geo-Beard) [inline db/data/test-data/playground.sql]: Remember to not include your playground files.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -68,11 +90,45 @@ describe("app", () => {
...
   });
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("responds with an array", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          const comments = body;
+          expect(Array.isArray(comments)).toBe(true);
+          expect(comments.length).toBeGreaterThan(0);
```
Comment (Geo-Beard) [inline __tests__/app.test.js]: You can specify the array length here as we know what it is from the test data.

File: __tests__/app.test.js [hunk 3]
```diff
@@ -68,11 +90,45 @@ describe("app", () => {
...
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: expect.any(Number),
+            });
+          })
+
+          expect(comments).toBeSortedBy('created_at', {
+            descending: true
+          })
+        });
```
Comment (Geo-Beard) [inline __tests__/app.test.js]: I would add another 200 test for an article that exists but it has no comments - to check that we get an empty array returned.

**Review by Geo-Beard — COMMENTED** [review]
Looks OK overall, just one test to add. Once you've implemented those changes feel free to merge and move on - just make sure you don't run into too many merge conflicts with your previous ticket! Any issues, open a HD :)
